.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,20 @@
-<!--
-Thank you for your interest in and contributing to Kibana! There
-are a few simple things to check before submitting your pull request
-that can help with the review process. You should delete these items
-from your submission, but they are here to help bring them to your
-attention.
-
-- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
-- Have you followed the [contributor guidelines](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md)?
-- If submitting code, have you included unit tests that cover the changes?
-- If submitting code, have you tested and built your code locally prior to submission with `yarn test && yarn build`?
-- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
--->
\ No newline at end of file
+### Summary
+
+Summarize your PR. If it includes design elements include a screenshot or gif.
```

Comment:
Contributors may not know how to characterize design elements. How about making this a bit broader?

> If it involves visual changes include a screenshot or gif.

Reply:
Good point. I'll change.